Generals/Code/Libraries/Source/WWVegas/WWLib/lcw.cpp

```diff
@@ -168,7 +168,7 @@ int LCW_Uncomp(void const * source, void * dest, unsigned long )
 }
 
 
-#if defined(_MSC_VER)
+#if defined(_MSC_VER) && defined(_M_IX86)
```

Comment:
Add `// defined(_MSC_VER) && defined(_M_IX86)` to #endif below, like in ZH edit.